Kernel unit tests fail if Opencast is running in the background

**Describe the bug**
Run Opencast in the background, then try and build `r/9.x`'s kernel module, and you'll see this:

```
[INFO] Running org.opencastproject.kernel.security.TrustedHttpClientImplTest                                                                                                                                       
15:49:04  WARN (TrustedHttpClientImpl:597) - URI http://localhost:8080/fake does not support digest authentication                                                                                                 
[ERROR] Tests run: 12, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 0.723 s <<< FAILURE! - in org.opencastproject.kernel.security.TrustedHttpClientImplTest                                                   
[ERROR] noDefaultHttpConnectionFactoryResultsInException(org.opencastproject.kernel.security.TrustedHttpClientImplTest)  Time elapsed: 0.278 s  <<< FAILURE!                                                       
java.lang.AssertionError                                                                                                                                                                                           
        at org.opencastproject.kernel.security.TrustedHttpClientImplTest.noDefaultHttpConnectionFactoryResultsInException(TrustedHttpClientImplTest.java:142)
```

--- a/modules/kernel/src/test/java/org/opencastproject/kernel/security/TrustedHttpClientImplTest.java
+++ b/modules/kernel/src/test/java/org/opencastproject/kernel/security/TrustedHttpClientImplTest.java
@@ -137,7 +137,7 @@ public class TrustedHttpClientImplTest {
   @Test
   public void noDefaultHttpConnectionFactoryResultsInException() {
     try {
-      client.execute(new HttpPost("http://localhost:8080/fakeEndpoint"));
+      client.execute(new HttpPost("http://localhost:9/fakeEndpoint"));
       // It should fail without a default http connection factory
       Assert.fail();
     } catch (TrustedHttpClientException e) {

